app/scripts/contentscript.js

```diff
@@ -73,6 +73,6 @@ function isAllowedSuffix (testCase) {
   if (doctype) {
     return doctype.name === 'html'
   } else {
-    return false
+    return true
```

Comment:
I'll try to make and add an example site URL  that opens the different situations in where iframes could be opened. Then you can check that they effectively don't send along a doctype. But I agree that this injection logic is there for a reason. If XML files actually send a doctype , they shouldn't be affected by this change tho.